hnn_core/network.py

```diff
@@ -162,6 +165,14 @@ class Network(object):
         If True, add drives as defined in the params-dict. NB this is mainly
         for backward-compatibility with HNN GUI, and will be deprecated in a
         future release. Default: False
+    inplane_distance : float
```

Comment:
would it make more sense to have a method called `set_cell_positions` rather than bake it into the `__init__` ? That way, say we read a network and want to change the in-plane separation later, we can do that.

Reply:
Hmm... I think the positions _have_ to be set in `init`, right? So would the method really help?

Reply:
> would it make more sense to have a method called `set_cell_positions`

I don't think so. Semantically, changing cell positions is equivalent to creating a new net, so I think the API should be to instantiate a new `net` with the spacing wanted. Note that in the future there may be other topologies with more parameters than just in-plane distance.

Pragmatically, `_create_cell_coords` in `init` returns the positions of the cells. These are used (still in `init`) by  `_add_cell_type` to set `net.pos_dict`.

Maybe I don't understand the API you're envisioning @jasmainak ?

Reply:
I guess I'm wondering how this would fit in with https://github.com/jonescompneurolab/hnn-core/issues/402

if you get an already instantiated network from the reader function, how do you modify the cell spacing afterwards?